pkg/runner/cluster_k8s.go

```diff
@@ -304,6 +304,9 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 		}
 	}
 
+	if !cfg.KeepService {
```

Comment:
Related to: https://github.com/testground/testground/issues/778 (+ http timeouts on daemon)